t/secret/aws.t

```diff
@@ -0,0 +1,321 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+BEGIN {
+    $ENV{AWS_REGION} = "us-east-1";
+    $ENV{AWS_ACCESS_KEY_ID} = "access";
+    $ENV{AWS_SECRET_ACCESS_KEY} = "secret";
+    $ENV{AWS_SESSION_TOKEN} = "token";
+}
+
+use t::APISIX 'no_plan';
+
+repeat_each(1);
+no_long_string();
+no_root_location();
+log_level("info");
+
+run_tests;
+
+__DATA__
+
```

Comment:
A test for checking that the input configuration conforms to the rules is needed.

ref: https://github.com/apache/apisix/blob/1ded3a837c52fd268927d76f6708553934b77a03/t/plugin/loki-logger.t#L35-L67